keras/layers/embeddings.py

```diff
@@ -135,7 +135,11 @@ def compute_output_shape(self, input_shape):
     def call(self, inputs):
         if K.dtype(inputs) != 'int32':
             inputs = K.cast(inputs, 'int32')
-        out = K.gather(self.embeddings, inputs)
+        # Use K.embedding(mx.sym.Embedding) in MXNet backend instead of K.gather(mx.sym.take)
```

Comment:
Please create a Github issue and link it from here.

Reply:
created: [here](https://github.com/awslabs/keras-apache-mxnet/issues/63)